### PR TITLE
July PyData London meet-up news.

### PR DESCRIPTION
**Pull request by mszeszo**

### CODE REVIEW

**Comment by mszeszo** [comment]
July 2024 PyData London meet-up slides.

File: 2024-07-02_PyData-News-July-2024/custom.css
Comment (kynan) [inline 2024-07-02_PyData-News-July-2024/custom.css:1]: This file doesn't appear to be used?

File: 2024-07-02_PyData-News-July-2024/styles/custom.css
Comment (kynan) [inline 2024-07-02_PyData-News-July-2024/styles/custom.css:1]: This file appears to be unused?

File: 2024-07-07_PyData-News-July-2024
Comment (kynan) [inline 2024-07-07_PyData-News-July-2024:1]: Please revert this file

**Review by kynan — CHANGES_REQUESTED** [review]

**Pull request merged**